README.md

```diff
@@ -1,2 +1,232 @@
-# learn-offline-first
+# Learn offline first
 Learn how to build web apps that can be used offline!
+
+## What?
+
+Offline first web applications are websites which make offline functionality a priority.
```

Comment:
@finnhodgkin & @ZooeyMiller we _prefer_ to limit our line length to 80 characters
to aid review/edit on smaller screens. 😉